db/seedfiles/translations/en/dc/shared.rb

```diff
@@ -44,7 +44,7 @@
     "en.has_eligible_health_coverage_from_hra_short_min" => "Does this person currently have access to health coverage or a Health Reimbursement Arrangement that they are not enrolled in (including through another person, like a spouse or parent)?",
     "en.hra_benefit" => "<b>What's the maximum self-only amount of reimbursement offered by this employer?</b> <br> <br>Employers that offer individual coverage HRAs must send written notices that include the maximum self-only reimbursement amount. If you have questions about the HRA, like what this amount is, look at the employer's HRA notice or check with the employer.",
     "en.support_texts.net_annual_income" => "<b>Annual Income for %{application_applicable_year}</b><br><br> This is the total of all income you told us about for this person minus any income adjustments for %{application_applicable_year}. If you think this amount is incorrect, select the pencil icon to go back to the application and make changes. If a certain type of income varies, you may want to estimate it for a longer period of time (for example, monthly or annually instead of weekly).",
-    "en.support_texts.relationship" => "<b>Relationship types</b><br><br> Select the option that best describes the family relationship. <ul><li>'Child' includes biological, adoptive, step, and foster children</li><li>'Parent' includes biological, adoptive, step, and foster parents</li><li>'Sibling' includes biological, half, adoptive, step, and foster siblings</li></ul><br> If these people are related but the type of relationship isn't on this list (for example, if you are cousins), select 'Unrelated'.",
+    "en.support_texts.relationship" => "<b>Relationship types</b><br><br> Select the option that best describes the family relationship. <ul><li>'Child' includes biological, adoptive, step, and foster children</li><li>'Parent' includes biological, adoptive, step, and foster parents</li><li>'Sibling' includes biological, half, adoptive, step, and foster siblings</li></ul><br> If these people are related but the type of relationship isn't on this list, select 'Unrelated'.",
```

Comment:
if we are already modifying this string, could we break it up to wear we don't need to leave the css in the translation?

Reply:
Discussion had offline: this change would require modifying a modal that handles single keys to be able to receive an array of keys and would not be worth it. Approving without change.